PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -0,0 +1,289 @@
+
+4. the asset owner documents the following:
+
+    a. PV asset type and brand
+
+    b. location of the installation (address, or GPS coordinates if an address is not available)
+
+    c. date of installation
+
+5. the power produced by the PV asset does not displace other on-site power generation (i.e. there is no on-site power generation whose output is affected by the generation of the PV asset)
+
+6. the asset owner or operator is not already being rewarded for self-consumption by another entity (e.g. government subsidies, ...)
+
+7. site demand (either as total demand or grid import) is measured by a digital meter with high resolution data, which can either be communicated automatically by digital means or retrieved manually
+
+In cases where power is exported from the site (i.e. not all solar generation is self-consumed), the following requirements shall also be met:
+
+8. the electricity export from the site is measured by a digital meter with high resolution data, which can either be communicated automatically by digital means or retrieved manually.
+
+### Requirements on the system during the project lifetime
+
+During the whole duration of the project, the PV asset owner shall demonstrate proof of ownership of the asset and continued operation of the self-consumption. This documentation shall be made via all of the following:
+
+- sharing of consumption data throughout the period,
+
+- in the case of usage of the environmental attribute in reporting:
+    - contractual renunciation to the possibility of transfer of the environmental assets to a third party.
+
+- in the case of sales of the environmental attribute to a third party:
+    - contractual transfer of the environmental attribute ownership of the self-consumed generation,
+    - carbon accounting for the site (location and market-based) should reflect the sale of the carbon reduction corresponding to the transfer of the certificate.
+
+
+### Requirements on the operating context
+
+The project country shall meet the following criteria:
+1. delivery of micro-solar is not a requirement for building development or to meet an enforced carbon quota,
+2. emission intensity data is available at the hourly (or sub-hourly) level for the national or local grid.
+
+In cases where this methodology is used to generate carbon credits, extra requirements shall be met:
+
+3. PV arrays are not installed on more than 10% of all buildings in the market of operation
+
+
+## Project boundary
+
+GHG sources included or excluded from the project boundary.
+
+| Source                     | Gas     | Included | Justification           |
+| -------------------------- |---------|----------|------------------------ |
+| Grid electricity displaced | $CO_2$  | Yes      | Main emission source    |
+| Grid electricity displaced | $NO_x$, $SO_x$  | No       | Emissions data is not sufficiently granular and available the half hour or hour |
+
```

Comment:
Have you thought about the emissions due to other life-cycle phases e.g. manufacturing, installation etc?

Reply:
Whether these are taken into account depends on the emission intensity data. ElectricityMaps uses lifecycle data (from IPCC) so that it would take this into account if you use that specific factor of theirs. Other providers typically take a fuel-based approach (i.e. more or less scope 1), which would not account for the lifecycle.
On this level, we're therefore bound by what's available for a given geography